store/storetest/user_store.go

```diff
@@ -658,7 +658,7 @@ func testUserStoreGetAllProfiles(t *testing.T, ss store.Store) {
 	})
 
 	t.Run("filter by system_user only", func(t *testing.T) {
-		actual, userErr := ss.User().GetAllProfiles(&model.UserGetOptions{
+		actual, _, userErr := ss.User().GetAllProfiles(&model.UserGetOptions{
 			Page:    0,
 			PerPage: 10,
 			Roles:   []string{"system_user"},
```

Comment:
Can we please add a testcase to verify the two new flags? Other things look good to me.